android/java/org/chromium/chrome/browser/crypto_wallet/util/AndroidUtils.java

```diff
@@ -98,4 +98,12 @@ private static void setViewVisibility(boolean isVisible, View... views) {
     public static int getScreenHeight() {
         return Resources.getSystem().getDisplayMetrics().heightPixels;
     }
+
+    /**
+     * Converts dip values to pixels on current device.
+     */
+    public static int dpToPx(Context ctx, int dp) {
```

Comment:
use chromium utils for that https://source.chromium.org/chromium/chromium/src/+/main:ui/android/java/src/org/chromium/ui/base/ViewUtils.java;l=155?q=ViewUtils.java&ss=chromium%2Fchromium%2Fsrc